lib/construct.gi

```diff
@@ -28,30 +28,46 @@ InstallMethod(CreateEmptyFile,[IsString],
     PrintTo(name,"");
 end);
 
+InstallMethod(InitPolymakeObject,[IsPolymakeObject],
+        function(poly)
+    local appstring;
+    # In polymake 4.1 this is required.
+    appstring:=Concatenation(
+                        "_application polytope\n",
+                        "_type Polytope\n\n"
+                  );
+    AppendToPolymakeObject(poly, appstring);
```

Comment:
I wonder if `InitPolymakeObject` is really necessary: it always follows a call to `CreateEmptyFile`, so it seems that combination could instead be replaced by a single call to `ClearPolymakeObject`, i.e., something like `ClearPolymakeObject(poly,["polytope", VERSION, "Polytope"]);` -- however, I have no idea what VERSION should be. (I guess one could always enhance `ClearPolymakeObject` to skip outputting the version if the middle component of the argument list is not bound).

Reply:
I did consider using ClearPolymakeObject, but I did not like adding an unnecessary and rather arbitrary version. If I had made version an optional parameter the resulting change would probably be bigger than mine.